workers/generic-worker/chain_of_trust.go

```diff
@@ -63,6 +62,7 @@ type ChainOfTrustData struct {
 type ChainOfTrustTaskFeature struct {
 	task           *TaskRun
 	ed25519PrivKey ed25519.PrivateKey
+	disabled       bool
```

Comment:
is it something that was present in docker worker? 
I can't find any other comments explaining this addition

Reply:
We discovered a small bug where if a task feature (chain of trust in this case) fails to `Start()` but still publishes sig/cert files, so we still need to explicitly get the feature to `Stop()`, which is how this new member is being used.

Reply:
all good, I was just missing the context for this change. Maybe adding a comment would be enough.